pcp/versions/next/unauthorized.md

```diff
@@ -0,0 +1,50 @@
+Unauthorized
+===
+
+When a broker declines to deliver a message because it fails authorization,
+it must send an *unauthorized* message back to the original message sender.
+
+A clinet may send a message to multiple recipients while it's only authorized
+to send that message to some of them:
+
+```
+    client A                    broker B                multiple clients
+       |                           |                           |
+       |         1 message         |                           |
+       |-------------------------->| 2                         |
+       |                           |                           |
+       |      3 unauthorized       |                           |
+       |<--------------------------|                           |
+       |                           |                           |
+       |                           |         4 message         |
+       |                           |-------------------------->|
+       |                           |                           |
+```
+
+Client A sends a message to possibly multiple clients through broker B (1).
```

Comment:
I don't know if it will be messy, but perhaps having two or three recipient clients C, D, E would be more clear.